### PR TITLE
03/user can post

### PR DESCRIPTION
**Pull request by MartaBia**
Added submit button and text box for posts.
Not tested as the existing test file does not test for the logout button

### CODE REVIEW

File: frontend/src/components/feed/Feed.js [hunk 1]
```diff
@@ -1,49 +1,55 @@
...
+  const post = () => {};
+
+  if (token) {
+    return (
+      <>
+        <h2>Posts</h2>
+        <button onClick={logout}>Logout</button>
+        <button onClick={post}>Post</button>
+        <input
+          placeholder="Write your post here"
```
Comment (abodian) [inline frontend/src/components/feed/Feed.js:38]: For some reason when I run the server and code, the input box doesn't actually let me type anything. I can't see what is wrong with the code with a glance but maybe test it on your side and see if when you type if it shows any text?

**Review by abodian — CHANGES_REQUESTED** [review]

File: frontend/src/components/feed/Feed.js (hunk 1 above)
Comment (MartaBia) [reply frontend/src/components/feed/Feed.js:38]: Yes, I did notice it, we're working on the functionality of the button, we can take that too and work on both.

**Review by abodian — APPROVED** [review]
All working

File: frontend/src/components/feed/Feed.js (hunk 1 above)
Comment (abodian) [reply frontend/src/components/feed/Feed.js:38]: All fixed now thank you

**Review by IAmNotAbbas — APPROVED** [review]

**Pull request merged**